docs/source/user_guide/config.rst

```diff
@@ -231,6 +238,23 @@ For example, you can customize your FiftyOne config at runtime as follows:
         show_progress_bars=True,
     )
 
+.. _configuring-mongodb-connection:
+
+Configuring a MongoDB connection
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+By default, FiftyOne is installed with it's own `mongod` database
```

Comment:
Question: how does the manual DB configuration now work for environments other than Apple silicon where we don't provide a prebuilt `fiftyone-db`?

The hollow `fiftyone-db` package was a nice idea for Apple silicon! Can/should that be extended for *all* unsupported environments so that everyone can always `pip install fiftyone`? The hollow package could be setup to print an error pointing to this section in the docs if it is ever asked to spin up the DB with no `database_uri`.

Reply:
The only wheels we are missing at this point are linux i686 and win32. It doesn't seem important to me to add them. They are sunset platforms.

Filling out more support for linux distributions could be done, but I'd like to leave it out here. Detecting if the default db should not be used (SUSE system), and whether or not `mongod` is available via an alternative `fiftyone-db-*` package is a separate concern (implementation wise).

I'll make an issue. And add to the docs here that we do not support 32 bit systems.

Reply:
How does the FiftyOne install process look for someone on a linux distribution like SUSE today?

After seeing how the Apple silicon experience was, I'm just a bit confused on how people on unsupported distributions were able to follow the instructions we used to provide to get running in the first place.

Reply:
They would have no issue installing `fiftyone`. `pip` does not distinguish linux distributions. I only mention SUSE because it is listed [here](https://www.mongodb.com/try/download/community). But for example, I am on Ubuntu 20.04 and the default 18.04 `fiftyone-db` binary works fine for me.

So, `mongod` might fail for a SUSE user. I need to check. We need to add a `fiftyone-db-suse` in that case.

Reply:
Ah I see. So basically everyone with a 64 bit machine can `pip install fiftyone` after this PR, but users in unsupported linux environments (for example) might encounter an error when trying to fire up the database.

But this PR reduces the priority for a `fiftyone-db-suse` package (for example) **IF** the logging message that occurs when the DB service fails to start points the user to your docs on manual DB setup